services/web/server/src/simcore_service_webserver/projects/projects_utils.py

```diff
@@ -56,6 +56,10 @@ def _replace_uuids(node):
         return node
 
     project_copy["workbench"] = _replace_uuids(project_copy.get("workbench", {}))
+    if "ui" in project_copy:
```

Comment:
For the moment this fix is enough but I would create an issue to define a "clone" function in the Pydantic project class that takes care of uuids
Something that extends over a model copy: https://pydantic-docs.helpmanual.io/usage/exporting_models/#modelcopy